tests/test_packaged_modules.py

```diff
@@ -92,3 +106,16 @@ def test_imagefolder_drop_labels(image_file, drop_labels):
         assert all(example.keys() == {"image", "label"} for _, example in generator)
     else:
         assert all(example.keys() == {"image"} for _, example in generator)
+
+
+@pytest.mark.parametrize("sampling_rate", [None, 16_000])
```

Comment:
this test is logically a bit strange as it checks two things: that you receive an error if `sampling_rate` is not provided and that a dataset instantiates correctly when `sampling_rate` is provided + **also** that it correctly generates examples. Is it reasonable to move correct generation checking to a separate test or is it ok as it is?

Reply:
sorry for publishing the comment now, i accidentally pressed the wrong button...